curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-pyramid-generator/660f505d02b2bd513a1c3468.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 660f505d02b2bd513a1c3468
+title: Step 102
+challengeType: 0
+dashedName: step-102
+---
+
+# --description--
+
+Your pyramid is no longer inverted. This is because you are adding new rows to the **end** of the array.
+
+Update your loop body to add new rows to the beginning of the array.
+
+# --hints--
+
+You should not use the `.push()` method of `rows`.
+
+```js
+const stripped = __helpers.removeJSComments(code);
+assert.notMatch(stripped, /\.push/);
+```
```

Comment:
I personally think we should remove this hint altogehter.

Because if we have a situation where the camper that stops the project before this step, takes a day or two off and comes back to resume, there is a good chance they will forget which array method to use here.

We have seen that play out for other projects on the forum, where they will take time between steps and forget what they learned previously. So if this if the first hint, that will be a frustrating experience. 

<img width="720" alt="Screenshot 2024-04-09 at 9 13 29 PM" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/67210629/20172b8e-46cd-4f91-ac40-c773bcae21a4">

either the hint text should be augmented to say "use the unshift method instead" or just remove this test and show the second hint here instead

````
You should use the `unshift` method of `rows`.

```js
const stripped = __helpers.removeJSComments(code);
assert.match(stripped, /\.unshift/);
```
````